index.js

```diff
@@ -46,6 +46,7 @@ function dep() {
 
   if (!dep) {
     execa.commandSync('curl -LO https://deployer.org/deployer.phar')
+    execa.commandSync('sudo chmod +x deployer.phar')
```

Comment:
Will it work with sudo command here?

Reply:
I don't know. I don't tested that solution because I did this action for testing

```
> name: Deploy
> on:
>   push:
>     branches: [ master ]
>   pull_request:
>     branches: [ master ]
>   workflow_dispatch:
> jobs:
>   deploy:
>     name: deploy to prod
>     runs-on: ubuntu-latest
>     steps:
>       - uses: actions/checkout@v1
>       - name: Setup PHP
>         uses: shivammathur/setup-php@v2
>         with:
>           php-version: 8.0
>       - name: SetUp Deployer
>         run: curl -LO https://deployer.org/deployer.phar && mkdir bin && mv deployer.phar bin/dep && sudo chmod +x bin/dep
>       - name: Deploy
>         uses: deployphp/action@master
>         with:
>           private-key: ${{ secrets.PRIVATE_KEY }}
>           known-hosts: ${{ secrets.KNOWN_HOSTS }}
>           dep: test
```

Reply:
@antonmedv Have you tested this solution?

Reply:
I tested it and the sudo command work